apps/web/playwright/reschedule.e2e.ts

```diff
@@ -275,23 +277,28 @@ test.describe("Reschedule Tests", async () => {
     await page.locator('[data-testid="confirm-reschedule-button"]').click();
     await expect(page).toHaveURL(/.*booking/);
   });
-
   test("Should load Valid Cal video url after rescheduling Opt in events", async ({
     page,
     users,
     bookings,
+    browser,
   }) => {
+    // eslint-disable-next-line playwright/no-skipped-test
+    test.skip(!process.env.DAILY_API_KEY, "DAILY_API_KEY is needed for this test");
```

Comment:
IF process.env.DAILY_API_KEY is undefined videoCallUrl will be an empty string which IMO default the purpose of this test